DOCS-3839: GELI-Encrypted (legacy) pools disks can be replaced (#1423)
truenas/documentation: content/CORE/CORETutorials/Storage/Disks/DiskReplace.md

This disk must be installed in the TrueNAS system and not part of an existing storage pool.
Any data on the replacement disk is wiped as part of the process.

{{< expand "Can I replace a disk in a GELI-encrypted (Legacy) pool?" "v" >}}

> Although GELI-encryption has been depreciated, TrueNAS will still implement GELI-encryption during a "GELI-Encrypted (Legacy) pool" disk replacement. That will continue for the remaining life of the pool.

Reviewer: ```suggestion
Although GELI encryption is deprecated, TrueNAS implements GELI encryption during a "GELI-Encrypted (Legacy) pool" disk replacement. After the replacement, TrueNAS uses GELI encryption for the lifetime of that pool.
```
